metadata-ingestion/src/datahub/sql_parsing/sqlglot_utils.py

```diff
@@ -163,6 +165,28 @@ def get_query_fingerprint(
     return get_query_fingerprint_debug(expression, dialect)[0]
 
 
+def try_format_query(expression: sqlglot.exp.ExpOrStr, dialect: DialectOrStr) -> str:
```

Comment:
I got briefly confused by the argument name `dialect`, as I saw you were passing in a platform name. Think this would be slightly clearer as `dialect_or_platform`

Reply:
I try to use platform as datahub platform name str, and dialect as the sqlglot dialect object

in this case, it'll most commonly be platform, so I've renamed it as such